src/api/app/helpers/webui/responsive_ux/actions_helper.rb

```diff
@@ -0,0 +1,11 @@
+module Webui::ResponsiveUx::ActionsHelper
+  def link_to_action(label, url, args)
```

Comment:
This is nice to avoid repetition, but for something as simple as a link with an icon and some text, I would be against introducing this change. I prefer repeating some simple code over an abstraction like this helper method.

Reply:
After thinking for a while I've decided to not use the helper method